sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -332,6 +333,7 @@ class DeviceBinaryImage {
   DeviceBinaryImage::PropertyRange CompositeSpecConstIDMap;
   DeviceBinaryImage::PropertyRange DeviceLibReqMask;
   DeviceBinaryImage::PropertyRange KernelParamOptInfo;
+  DeviceBinaryImage::PropertyRange SyclIsEsimdImage;
```

Comment:
It looks like a backward incompatible change to me as this class crosses library boundaries.
Can we instead of adding a new field to the class search for `__SYCL_PI_PROPERTY_SET_SYCL_IS_ESIMD_IMAGE` in the `Bin` in the `getSyclIsEsimdImage` method(like it's done in the `PropertyRange::init` method)?

tagging @kbobrovs

Reply:
In fact, I don't need `SyclIsEsimdImage` to be a member of the `DeviceBinaryImage` class. Fixed.